src/verinfast/cloud/gcp/instances.py

```diff
@@ -6,19 +6,25 @@
 
 from google.api_core.exceptions import NotFound
 from google.cloud import compute_v1
-from google.cloud.monitoring_v3 import Aggregation, MetricServiceClient, TimeInterval, ListTimeSeriesRequest  # noqa: E501
+from google.cloud.monitoring_v3 import (
+    Aggregation,
+    MetricServiceClient,
+    TimeInterval,
+    ListTimeSeriesRequest,
+)  # noqa: E501
 
 from verinfast.cloud.gcp.zones import zones
-from verinfast.cloud.cloud_dataclass import \
-    Utilization_Datapoint as Datapoint,  \
-    Utilization_Datum as Datum
+from verinfast.cloud.cloud_dataclass import (
+    Utilization_Datapoint as Datapoint,
+    Utilization_Datum as Datum,
+)
 
 # os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = "/Users/jason/.config/gcloud/application_default_credentials.json"  # noqa: E501
```

Comment:
it's a quote, within a quote. yodawg.gif

Reply:
and this is a comment in response to a comment about a comment within a comment. headsplode.gif